### PR TITLE
Clean newlines

### PR DESCRIPTION
**Pull request by constancebello**
Remove leading newlines in files and replace lines containing only whitespace with blank lines

### CODE REVIEW

**Comment by steveklabnik** [comment]
Thank you!

File: src/test/run-pass/lexer-crlf-line-endings-string-literal-doc-comment.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ignore-tidy-cr ignore-license
 // ignore-tidy-cr (repeated again because of tidy bug)
 // license is ignored because tidy can't handle the CRLF here properly.
-
+
```
Comment (retep998) [inline src/test/run-pass/lexer-crlf-line-endings-string-literal-doc-comment.rs]: No! Not this file! This test is specifically supposed to use crlf! This is changing the blank lines from \r\n to \n!

Comment (steveklabnik) [inline src/test/run-pass/lexer-crlf-line-endings-string-literal-doc-comment.rs]: Ahhhhh eff. @aarzee can you send a PR fixing this?

**Comment by steveklabnik** [comment]
If @aarzee can't do this soon, I will fix it tomorrow. Of course I missed one :-1:

**Comment by aarzee** [comment]
Sorry, didn't realize this was wrong. I don't think I'll be able to fix this tonight/tomorrow as I've got a couple obligations involving limited Internet access. :(

**Comment by steveklabnik** [comment]
No worries! I'll do it right now then.

**Comment by steveklabnik** [comment]
Sent: https://github.com/rust-lang/rust/pull/28876

**Pull request merged**